cogs/Staff.py

```diff
@@ -148,9 +148,9 @@ async def unban(
         await sucEmb(inter, "User has been unbanned successfully!")
 
     @commands.slash_command(
-        name="mute", description="Timeout(mute)'s a user from the server."
+        name="mute", description="'timeout's a user from the server."
     )
-    @commands.has_role("Staff")
```

Comment:
why... this is Counterintuitive...

Reply:
we wanted to make the bot global, and staff role is very proprietary

Reply:
requiring manage roles perm is NOT the way to achieve this

Reply:
thats not a guarantuee or it forces them to create a rule wich just complicates stuff, manage roles perm was used because mutes used to be a muted role wich cant speak and if the person has that permission they could mute people like that, I'll look into it if there is a timeout permission

Reply:
Except it's not just used for mute. It's used for everything

Reply:
the only place where i feel its out of place is in the warning commands but there isn't a more suitable role for that so I think it's fine, it being used for reaction based roles makes sense I think.

Reply:
manage users does that exist? 

> the only place where i feel its out of place is in the warning commands but there isn't a more suitable role for that so I think it's fine, it being used for reaction based roles makes sense I think.

Reply:
moderate_members and thats used for timeout perms so I added that to mute, could be used for warnings aswell, what do you say?